guides/source/testing.md

```diff
@@ -580,11 +579,11 @@ To test AJAX requests, you can specify the `xhr: true` option to `get`, `post`,
 `patch`, `put`, and `delete` methods:
 
 ```ruby
-test "ajax request responds with no layout" do
```

Comment:
It doesn't render a layout but I think we shouldn't be encouraging others to test for it.